docs/service_specs/submission.md

```diff
@@ -18,44 +18,32 @@ An example of the submission package can be found at `\link-cloud\Submission Exa
 - **Technology**: .NET Core
 - **Image Name**: link-submission
 - **Port**: 8080
-- **Database**: MSSQL (previously Mongo)
+- **Database**: MongoDB
```

Comment:
This is because the DotNet/Submission project should be removed, it is out-dated, and replaced by Java/Submission. I plan to create a separate PR for removing the DotNet/Submission project.

Reply:
Whoops, I was wrong about this. I was thinking of the validation service, instead of the submission service.

Reply:
The service is using MongoDB, and is starting up registering with `builder.Services.Configure<MongoConnection>(builder.Configuration.GetRequiredSection(SubmissionConstants.AppSettingsSectionNames.Mongo));`
I am removing the `DatabaseProvider` property from `appsettings.json` and changing the docs for the service to reference MongoDB, instead of SQL Server.